pyproject.toml

```diff
@@ -18,7 +29,81 @@ xcp-ng-dev-env-create = "xcp_ng_dev.cli:build"
 Homepage = "https://github.com/xcp-ng/xcp-ng-build-env/"
 
 [build-system]
-requires = ["setuptools >= 77.0.3", "setuptools-scm>=8"]
+requires = ["setuptools >= 77.0.3", "setuptools-scm>=8", "wheel"]
```

Comment:
why is this needed?

I did not see that in official examples though they do build wheels, and https://github.com/xcp-ng/xcp-ng-build-env/pull/49 seems to show it works without

Reply:
@psafont may know better than me :slightly_smiling_face: